pkgs/development/python-modules/mlt-bindings/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, mlt, python, buildPythonPackage, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "mlt-bindings";
+  version = mlt.version;
+  disabled = pythonOlder "3.7";
+
+  src = "${mlt}/lib/python${python.sourceVersion.major}.${python.sourceVersion.minor}/site-packages";
+
+  nativeBuildInputs = [ mlt ];
+
+  doCheck = false;
```

Comment:
Please add a comment why you disabled tests.

Reply:
Removed completely after refactoring